_projects/_template.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Example Astropy
+desc: This is the first idea for astropy
```

Comment:
This is a one line description - it appears on the box together with the title. Added a description in the comment.